apps/main/src/app/.well-known/walletconnect.txt/route.ts

```diff
@@ -2,8 +2,8 @@ import { headers } from 'next/headers'
 
 const WALLET_CONNECT_ACCOUNT = `c3fe8dd8-93df-44af-803f-83798aa1d440`
 
-// for curve-dapp-git-feat-connect-modal-curvefi.vercel.app, other domains can be added the dashboard
-const VERCEL_DOMAIN_VERIFICATION = '0c282d3503877d627a6a75095ebdb60530b900d3b194d046f6d8da3ae19fd48e'
+// for curve-dapp-git-fix-wallet-connect-curvefi.vercel.app, other domains can be added the dashboard
```

Comment:
did you remove the old domains? usually the token stays the same between preview URLs

Reply:
Oh, I did yes 🤔